lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -207,6 +206,14 @@ def database_yaml
           nil
         end
 
+        def load_yaml(src)
+          if Psych::VERSION > '4.0'
```

Comment:
Based on https://github.com/rubocop/rubocop-rails/pull/516 link, So I'm not sure `Psych::VERSION > 4.0` is better than `respond_to?(:unsafe_load_file)`.